test/token/ERC20/extensions/ERC4626.test.js

```diff
@@ -88,7 +88,7 @@ contract('ERC4626', function (accounts) {
       const sharesForDeposit = await vault.previewDeposit(value, { from: holder });
       const sharesForReenter = await vault.previewDeposit(reenterValue, { from: holder });
 
-      // Do deposit normally, triggering the _beforeTokenTransfer hook
+      // Do deposit normally, triggering the _update hook
```

Comment:
The order is important here so I'd do:

```suggestion
      // Deposit normally, reentering before the internal `_update`
```